boards/sim/sim/sim/scripts/Make.defs

```diff
@@ -209,6 +209,7 @@ endif
 
 LDELFFLAGS = -r -e main --gc-sections
 LDELFFLAGS += -T $(call CONVERT_PATH,$(BOARD_DIR)$(DELIM)scripts$(DELIM)gnu-elf.ld)
+LDFLAGS += -Wl,--gc-sections
```

Comment:
Do we need to remove `--gc-sections` from `LDELFFLAGS`?

Reply:
LDFLAGS is different with LDELFFLAGS, LDELFFLAGS used in loadable applications